apps/server/src/shared/domain/service/board-copy.service.spec.ts

```diff
@@ -145,7 +145,9 @@ describe('board copy service', () => {
 				const user = userFactory.build();
 
 				lessonCopyService.copyLesson.mockReturnValue({
-					status: { title: originalLesson.name, type: CopyElementType.LESSON, status: CopyStatusEnum.NOT_IMPLEMENTED },
+					title: originalLesson.name,
+					type: CopyElementType.LESSON,
+					status: CopyStatusEnum.NOT_IMPLEMENTED,
 				});
```

Comment:
also the copy entity is missing here right? and with that the tests for adding that copy to the references of the board are missing